CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## master / unreleased
 
+* [FEATURE] Introduced `ruler.for-outage-tolerance`, Max time to tolerate outage for restoring "for" state of alert. #2783
+* [FEATURE] Introduced `ruler.for-grace-period`, Minimum duration between alert and restored "for" state. This is maintained only for alerts with configured "for" time greater than grace period. #2783
+* [FEATURE] Introduced `ruler.for-resend-delay`, Minimum amount of time to wait before resending an alert to Alertmanager. #2783
```

Comment:
Uups! The flag is `ruler.resend-delay`. @owen-d Would you mind opening a PR to fix the CHANGELOG entry, please?

Reply:
Also put it in the correct place, like we ask in the checklist.